Cinebase/Gateways/WebAPI/Parsing/MovieParser.swift

```diff
@@ -20,19 +20,23 @@ class MovieParser: ModelParser {
     
     func parse(fromJSON json: [String : Any]) -> Result<Movie> {
  
+        let dateParser = DateParser()
+        
         guard let movieID = json["id"] as? Int,
             let title = json["title"] as? String,
             let imagePath = json["poster_path"] as? String,
             let backdropPath = json["backdrop_path"] as? String,
-            let overview = json["overview"] as? String else {
+            let overview = json["overview"] as? String,
+            let releaseDateString = json["release_date"] as? String,
+            let releaseDate = dateParser.parse(from: releaseDateString) else {
                 let parsingError = ParsingError<Movie>(json: json)
                 return .failure(parsingError)
         }
```

Comment:
It will be better to put a default value as ND if there is a error in data parsing, it can be a case where there is a knowledge of future movie but not a specific date of release